SchedulerCreator/src/js/exporters/HTML/HTMLExporter.js

```diff
@@ -34,6 +35,18 @@ module.exports = (timedScheduleObject) => {
                     if (theClass.teacher) {
                         classCode += `<span class="teacher">${theClass.teacher}</span>`;
                     }
+                    if (theClass.color) {
+                        if(colors[i]) {
+                            document.getElementsByClassName("class").style.backgroundColor = colors
```

Comment:
DOM API not available within Node.js

Reply:
Look at the new file change that occurred after this